src/components/SignUp/SignUpInput.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Field, ErrorMessage } from "formik";
+
+const SignUpInput = ({ type, name, children }) => {
+  return (
+    <div className="mb-2 col-12 col-md-6">
+      <label htmlFor={name} className="d-block mb-2 p-0">
+        {children}
+      </label>
+      <Field type={type} id={name} name={name} className="w-100"/>
```

Comment:
Use controlled component (pass a **ref** to the form to be able to access the values of all inputs from it or use [Formik hooks](https://formik.org/docs/api/useFormik))